fedora/README.md

```diff
@@ -26,7 +26,7 @@ The metalink `http://mirrors.fedoraproject.org` is used to automatically select
 
 # Supported Docker versions
 
-This image is officially supported on Docker version 1.6.2.
+This image is officially supported on Docker version 1.6.2 and later.
```

Comment:
umm, docker on rawhide is already 1.7.0, and by officially I guess I meant fedora, but maybe you mean docker :|